contract_compassion/models/contracts.py

```diff
@@ -222,10 +222,16 @@ def _compute_contract_duration(self):
     def create(self, vals):
         if 'commitment_number' not in vals:
             partner_id = vals.get('partner_id')
+            correspondent_id = vals.get('correspondent_id')
```

Comment:
This should be done in module `sponsorship_compassion` where the field `correspondent_id` is defined.